test/helpers/helpers.go

```diff
@@ -163,20 +163,12 @@ func CreateOperatorSourceDefinition(namespace string) *marketplace.OperatorSourc
 		},
 	}
 }
-
-// CheckCatalogSourceConfigAndChildResourcesCreated checks that a CatalogSourceConfig
-// and it's child resources were deployed.
-func CheckCatalogSourceConfigAndChildResourcesCreated(client test.FrameworkClient, cscName string, namespace string, targetNamespace string) error {
-	// Check that the CatalogSourceConfig was created.
-	resultCatalogSourceConfig := &marketplace.CatalogSourceConfig{}
-	err := WaitForResult(client, resultCatalogSourceConfig, namespace, cscName)
-	if err != nil {
-		return err
-	}
-
+// CheckCatalogSourceConfigChildResourcesCreated checks that a CatalogSourceConfig's
+// child resources were deployed.
+func CheckCatalogSourceConfigChildResourcesCreated(client test.FrameworkClient, cscName string, namespace string, targetNamespace string) error {
```

Comment:
If anyone has any suggestions on how to further distinguish `CheckCatalogSourceConfigChildResourcesCreated` from `CheckCatalogSourceConfigAndChildResourcesCreated` please share.